### DISCUSSION TITLE
[feature request] disallow repositories from blocked users showing up in the "explore repositories" sidebar

### DISCUSSION THREAD

**Discussion by ghost** [community/community#3047] [Discussions] [answered]
i understand this isn't the place to make this request; however, i originally made this feature request in https://github.com/github/explore/issues/1789, but it seems all the issues have been deleted since issues were disabled in the repository

i've faced harassment (offsite) from a number of users in a particular community. they've all been blocked, but due to my past involvement in the community, github continues to recommend their repositories to me; it's pretty annoying at the best of times, somewhat upsetting at other times

it would be really nice to have those users not show up in that sidebar

**Comment by evi-liu** [comment, marked as answer]
Thanks for the feedback! I've passed it along to our team working on Explore.
